Netkan/Sources/Github/IGithubApi.cs

```diff
@@ -2,6 +2,7 @@ namespace CKAN.NetKAN.Sources.Github
 {
     internal interface IGithubApi
     {
+        GithubRepo GetRepo(GithubRef reference);
         GithubRelease GetLatestRelease(GithubRef reference);
```

Comment:
@politas GetRepo is made available here. IGithubApi gets passed in during the object construction of GitHubTransformer.